influxdb3/src/help/serve_all.txt

```diff
@@ -22,253 +23,261 @@ Examples:
-                                    [env: INFLUXDB3_ADMIN_TOKEN_RECOVERY_HTTP_BIND_ADDR]
-  --admin-token-file <FILE>        File containing offline admin tokens. Used for automated
-                                   deployments where the admin token is pre-generated and loaded at startup.
-                                    [env: INFLUXDB3_ADMIN_TOKEN_FILE=]
-
+  --http-bind <ADDR>                           Address for HTTP API requests [default: 0.0.0.0:8181]
+                                                 [env: INFLUXDB3_HTTP_BIND_ADDR=]
+  --log-filter <FILTER>                        Logs: filter directive
+                                                 [default: info,iox_query::query_log=warn,influxdb3_query_executor::enterprise=warn]
+                                                 [env: LOG_FILTER=]
 
 {}
-  --data-dir <DIR>                 Location to store files locally [env: INFLUXDB3_DB_DIR=]
-  --bucket <BUCKET>                Bucket name for cloud object storage [env: INFLUXDB3_BUCKET=]
-  --gen1-duration <DURATION>       Duration for Parquet file arrangement [default: 10m]
-                                    [env: INFLUXDB3_GEN1_DURATION=]
-  --gen1-lookback-duration <DURATION>
-                                   The amount of time that the server looks back on startup
-                                   when populating the in-memory index of gen1 files.
-                                    [env: INFLUXDB3_GEN1_LOOKBACK_DURATION=]
-  --delete-grace-period <DURATION> Grace period for hard deleted databases and tables before they are
-                                   removed permanently from the catalog [default: 24h]
-                                    [env: INFLUXDB3_DELETE_GRACE_PERIOD=]
-  --retention-check-interval <INTERVAL>
-                                   The interval at which retention policies are checked and enforced.
-                                   Enter as a human-readable time, e.g., '30m', '1h', etc. [default: 30m]
-                                    [env: INFLUXDB3_RETENTION_CHECK_INTERVAL=]
+  --tls-key <KEY_FILE>                         The path to the key file for TLS to be enabled
+                                                 [env: INFLUXDB3_TLS_KEY=]
+  --tls-cert <CERT_FILE>                       The path to the cert file for TLS to be enabled
+                                                 [env: INFLUXDB3_TLS_CERT=]
+  --tls-minimum-version <VERSION>              The minimum version for TLS. Valid values are
+                                               tls-1.2 and tls-1.3, default is tls-1.2
+                                                 [env: INFLUXDB3_TLS_MINIMUM_VERSION=]
+  --without-auth                               Run InfluxDB 3 server without authorization
+  --disable-authz <RESOURCES>                  Optionally disable authz by passing in a comma separated
+                                               list of resources. Valid values are health, ping, and metrics.
```

Comment:
i kinda like an indent if the help text is more than one line. just a nit

Reply:
no; i don't mean the wrapping that gh is showing. I mean that `Optionally ...` and the line `list of ...` have the same indent. I prefer the second line and more lines to be indented 2 spaces over. 

its just a preference.